packages/compiler/plugins/plugin-runner/src/runPlugin.ts

```diff
@@ -9,44 +11,39 @@ const DOCKER_PATH_TO_IR = path.join(DOCKER_FERN_DIRECTORY, "ir.json");
 
 export declare namespace runPlugin {
     export interface Args {
-        imageName: string;
+        pluginInvocation: PluginInvocation;
         pathToIr: string;
-        pluginConfig: unknown;
         pathToWriteConfigJson: string;
-        /**
-         * this directory will be mapped to DOCKER_CODEGEN_OUTPUT_DIRECTORY.
-         * the plugin will be instructed to write any files to this directory.
-         */
-        pluginOutputDirectory: string;
-        outputPathRelativeToRootOnHost: string | undefined;
+        relativeWorkspacePath: string | undefined;
     }
 }
 
 export async function runPlugin({
-    imageName,
+    pluginInvocation,
     pathToIr,
-    pluginConfig,
     pathToWriteConfigJson,
-    pluginOutputDirectory,
-    outputPathRelativeToRootOnHost,
+    relativeWorkspacePath,
 }: runPlugin.Args): Promise<void> {
-    const configJson = {
-        outputPathRelativeToRootOnHost,
+    const config: PluginConfig = {
+        relativeWorkspacePathOnHost: relativeWorkspacePath,
```

Comment:
we need this to be `outputPathRelativeToRootOnHost`

Reply:
Oops thanks

Reply:
wait those are different things

Reply:
oh i'm tracking I think

Reply:
yup they are different things -- was incorrect on what i needed at first